packages/user/src/migrations/Migration20240201081925.ts

```diff
@@ -0,0 +1,13 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20240201081925 extends Migration {
+
+  async up(): Promise<void> {
+    this.addSql('create table "user" ("id" text not null, constraint "user_pkey" primary key ("id"));');
```

Comment:
comment: will probably change a lot later, but not to forget the if exists and dissociate the table creation and the columns

Reply:
yes, will update the table definitions in a subsequent pr (including `if exists`)

Reply:
maybe worth leaving the migration out entirely for now